publication-commons/src/main/java/no/unit/nva/publication/model/business/Resource.java

```diff
@@ -225,6 +227,12 @@ public Resource fetch(ResourceService resourceService) throws NotFoundException
         return resourceService.getResourceByIdentifier(this.getIdentifier());
     }
 
+    // TODO: Implementation in this method should be used every place we fetch resource and publication after we have
+    //  migrated files
+    public Optional<Resource> fetchResourceWithFiles(ResourceService resourceService) {
+        return resourceService.getResourceAndFilesByIdentifier(this.getIdentifier());
+    }
+
```

Comment:
Will rename this method to "fetch" once we have migrated files, so fetching Resource will always serve files.